src/actions.ts

```diff
@@ -17,13 +17,9 @@ export default class ActionCreator extends BaseActionCreator {
 
   static readonly LOG_IN = "LOG_IN";
 
-  csrfToken: string;
-
-  constructor(fetcher?: DataFetcher, csrfToken?: string) {
+  constructor(fetcher?: DataFetcher) {
```

Comment:
Are you sure you don't need the `csrfToken`? This is something I'm not too familiar with and I'd ask Amy for help regarding making safe calls to the server for this.